rsp_vision/dashboard/pages/helpers/data_loading.py

```diff
@@ -0,0 +1,54 @@
+import pickle
+from pathlib import Path
+
+
+def load_data(store: dict) -> dict:
+    """This method loads the data from the pickle file.
+
+    Parameters
+    ----------
+    store : dict
+        The store object.
+
+    Returns
+    -------
+    dict
+        The data from the pickle file.
+    """
+    path = (
+        Path(store["path"])
```

Comment:
could `store["path"]` be set as Path when first entered and assumed to be path from thereon?

Reply:
I am not sure to understand what do you mean 🤔 you want the path to be saved locally? or the method to accept a generic `path` argument?